plugins/fireeye_hx/unit_test/test_quarantine_host.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import os
+
+from unittest import TestCase
+from icon_fireeye_hx.actions.quarantine_host import QuarantineHost
+from icon_fireeye_hx.actions.quarantine_host.schema import Input, Output
+from unit_test.util import Util
+from unittest.mock import patch
+from parameterized import parameterized
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+sys.path.append(os.path.abspath("../"))
```

Comment:
```suggestion
import sys
import os

from unittest import TestCase

sys.path.append(os.path.abspath("../"))

from icon_fireeye_hx.actions.quarantine_host import QuarantineHost
from icon_fireeye_hx.actions.quarantine_host.schema import Input, Output
from unit_test.util import Util
from unittest.mock import patch
from parameterized import parameterized
from insightconnect_plugin_runtime.exceptions import PluginException
```

Adding `../` path to sys.path should be done before other relative imports. Otherwise unit_tests doesn't work for me